domain-ee/ee-ep-merge-app/tests/test_api.py

```diff
@@ -1,6 +1,101 @@
+import uuid
+
+import pytest
 from fastapi.testclient import TestClient
+from model.merge_job import JobState
+from src.python_src.api import job_store
+
+MERGE = "/merge"
 
 
 def test_health(client: TestClient):
     response = client.get("/health")
     assert response.status_code == 200
+
+
+@pytest.fixture(autouse=True)
+def _job_store():
+    job_store.clear()
+
+
+@pytest.mark.parametrize("req", [
```

Comment:
Are these tests configured to automatically run with a command like ./gradlew :domain-ee:ee-ep-merge-app:test, i know there are some specific changes needed to build.gradle to automatically detect Python tests like this.

Reply:
Yes! That gradlew command still runs these parameterized tests!